--- Source/appfat.cpp
+++ Source/appfat.cpp
@@ -152,7 +152,9 @@ void InsertCDDlg()
            1024,
            "Unable to open main data archive (diabdat.mpq or spawn.mpq).\n"
            "\n"
-           "Make sure that it is in the game folder and that the file name is in all lowercase.");
+           "Make sure that it is in the game folder and that the file name is in all lowercase.\n"
+           "Please copy the file diabdat.mpq from your Diablo game disc or from the GOG \n"
+           "installer to the directory ~/snap/devilutionx/common/");
 
        UiErrorOkDialog("Data File Error", text);
        app_fatal(NULL);
